HW-1/task4.py

```diff
@@ -0,0 +1,26 @@
+#Программа загадывает число от 0 до 1000. Необходимо угадать число за 10 попыток.
+# Программа должна подсказывать «больше» или «меньше» после каждой попытки.
+# Для генерации случайного числа используйте код:
+# from random import randint
+# num = randint(LOWER_LIMIT, UPPER_LIMIT)
+
+from random import randint
+LOWER_LIMIT = 0
+UPPER_LIMIT = 100
+COUNT_TRY = 10
+RAND_NUMBER = randint(LOWER_LIMIT, UPPER_LIMIT)
+is_win = True
+for _ in range(COUNT_TRY + 1):
+    number = int(input('Введи число от 0 до 1000: '))
+    if number > RAND_NUMBER:
+        print('Ваше число БОЛЬШЕ загаданного')
+        is_win = False
+    elif number < RAND_NUMBER:
+        print('Ваше число МЕНЬШЕ загаданного')
+        is_win = False
+    elif number == RAND_NUMBER:
+        print('Вы выиграли!')
+        is_win = True
```

Comment:
Если число угадано угадано, то в дальнейших итерациях нет никакого смысла. 
```suggestion
        is_win = True
        break
```
Но переменная `is_win` тоже не нужна